charms/istio-pilot/src/charm.py

```diff
@@ -0,0 +1,340 @@
+                                        "content": env.get_template('mesh').render(tconfig),
+                                    },
+                                    {
+                                        "path": "meshNetworks",
+                                        "content": env.get_template('meshNetworks').render(tconfig),
+                                    },
+                                    {
+                                        "path": "values.yaml",
+                                        "content": env.get_template('values.yaml').render(tconfig),
+                                    },
+                                ],
+                            },
+                            {
+                                "name": "local-certs",
+                                "mountPath": "/var/run/secrets/istio-dns",
+                                "emptyDir": {"medium": "Memory"},
+                            },
+                            {
+                                "name": "inject",
+                                "mountPath": "/var/lib/istio/inject",
+                                "files": [
+                                    {
+                                        "path": "config",
+                                        "content": env.get_template('config').render(tconfig),
+                                    },
+                                    {
+                                        "path": "values",
+                                        "content": env.get_template('values').render(tconfig),
+                                    },
+                                ],
+                            },
+                        ],
+                    },
+                ],
+            },
+            k8s_resources={
+                "kubernetesResources": {
+                    "customResourceDefinitions": [
+                        {"name": crd["metadata"]["name"], "spec": crd["spec"]}
+                        for crd in yaml.safe_load_all(Path("files/crds.yaml").read_text())
+                    ],
+                    'customResources': custom_resources,
+                    "mutatingWebhookConfigurations": [
+                        {
+                            "name": "sidecar-injector",
+                            "webhooks": [
+                                {
+                                    "name": "sidecar-injector.istio.io",
+                                    "clientConfig": {
+                                        "service": {
+                                            "name": service_name,
+                                            "namespace": namespace,
+                                            "path": "/inject",
+                                            "port": config.get('webhook-port'),
+                                        },
+                                        #  "caBundle": ca_bundle,
```

Comment:
Is this supposed to be commented out?

Reply:
This was already commented out in the reactive code, I'll just remove it, unless you remember the context of this remaining in there.

Reply:
The context was probably that I forgot to remove it :smile: